Bigger.CrudCars.WebApi/Cars/CarController.cs

```diff
@@ -0,0 +1,56 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Bigger.CrudCars.Application.Cars;
+using Bigger.CrudCars.WebApi.Cars.Requests;
+using Bigger.CrudCars.Domain;
+
+namespace Bigger.CrudCars.WebApi.Cars
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CarController : ControllerBase
+    {
+        private readonly CarService _carService;
+
+        public CarController(CarService carService)
+        {
+            _carService = carService;
+        }
+        
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Car>>> GetAll()
+        {
+            var cars = await _carService.GetAll();

```

Comment:
```suggestion
            return Ok(await _carService.GetAll());
```
You can also do this if the cars variable doesnt need any modifications